detekt-rules/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/bugs/ImplicitDefaultLocaleSpec.kt

```diff
@@ -25,7 +31,64 @@ class ImplicitDefaultLocaleSpec : Spek({
                 fun x() {
                     String.format(Locale.US, "%d", 1)
                 }"""
-            assertThat(subject.compileAndLint(code)).isEmpty()
+            assertThat(subject.compileAndLintWithContext(wrapper.env, code)).isEmpty()
+        }
+
+        it("report String.toUpperCase() call without explicit locale") {
```

Comment:
To be compliant with the other test case descriptions in this file and in this repository.

```suggestion
        it("reports String.toUpperCase() call without explicit locale") {
```